LabTools/CoreWidgets/LoadPlotWidget.py

```diff
@@ -237,11 +239,16 @@ def create_plw(parent, load_fname=None):
         else:
             print("No headers present:",labels)
 
+        if 'data' in labels.keys():
+            parent.widgets['userDataWidget'].set_user_data_parse(labels['data'])
+
         # update the name information in the widget
         parent.widgets["loadPlotWidget"].load_file_name(load_fname)
 
         # store the hdr is there is more than one file
         parent.loaded_data_header[load_fname] = labels['hdr']
+        if SET_OUTPUT_TEXT:
```

Comment:
To do [defensive programming](https://en.wikipedia.org/wiki/Defensive_programming) you could specify `SET_OUTPUT_TEXT is True`